### ISSUE TITLE
[feature] Token holders

### ISSUE THREAD

**Issue by vncoelho** [CityOfZion/neo-scan#375]
Hey, @adrienmo and @lucianoengel,

I think that we are talked about this some time ago, but I do not remember if it was already included in the future direction.

Recently, I have been giving a seminar talking about "Transparency as a contribution to the end of inequalities".
The point that border me is that sometimes I am often showing etherscan.io instead of our blockexplorer.
The main reason is because I want to show to the audience the tokens percentage of holders and etc...
I think that we should move towards these kind of information as soon as possible.

What do you think?

**Comment by adrienmo** [comment]
@vncoelho I agree with you, could you detail the metrics you would like to have?

**Comment by vncoelho** [comment]
@adrienmo, I think that search for the list of most healthy address is enough (for NEO,GAS and NEPs5), because most of the other features of etherscan.io you already have.

![image](https://user-images.githubusercontent.com/17395762/48340404-afbdef80-e651-11e8-9ed6-4144fd0660c4.png)

![image](https://user-images.githubusercontent.com/17395762/48340437-c49a8300-e651-11e8-9c9d-aeafd3ca45ce.png)
